src/test/resources/features/checksum_verification.feature

```diff
@@ -61,7 +61,8 @@ Feature: Verify checksums
 		And the candidate "grails" version "1.3.9" is installed
 		And the response headers file is created for candidate "grails" and version "1.3.9"
 		And the exit code is 0
-
+		
+	@manual
```

Comment:
It seems to be impossible to build this locally. Am I the only one?

I'll be happy to bring this back if this is only occurring on my machine.

/cc @hgeraldino

Reply:
I'll take a look

Reply:
Thank you!